### ISSUE TITLE
본 프로젝트가 gpu 연산을 지원하는지 궁금합니다.

### ISSUE THREAD

**Issue by songchungjin** [hccho2/Tacotron2-Wavenet-Korean-TTS#3]
올려주신 코드와 손석희 샘플 데이터로

train 진행시 제 컴퓨터 환경에서는 1 step에 약 10초정도 걸립니다.

현재 50000 step 이상 진행을 하려면 약 7일가량 걸리는데

gpu 연산을 통해 빠르게 학습하고 싶어서
tensorflow-gpu  1.13.1 
cuda 10.0 
cudnn 7.6.0
환경에서 train 파일을 실행시키니

다음과 같은 에러가 발생합니다.

2019-06-10 17:33:39.138394: E tensorflow/stream_executor/cuda/cuda_dnn.cc:334] Could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR

2019-06-10 17:33:39.139695: E tensorflow/stream_executor/cuda/cuda_dnn.cc:334] Could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR

Exiting due to exception: Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above.
	 [[node model/Encoder/Encoder_0/conv1d/Conv2D (defined at /home/pirl/Tacotron2-Wavenet-Korean-TTS-master/tacotron2/tacotron2.py:82) ]]
	 [[node model/optimizer/Adam/update (defined at /home/pirl/Tacotron2-Wavenet-Korean-TTS-master/tacotron2/tacotron2.py:298) ]]

Caused by op 'model/Encoder/Encoder_0/conv1d/Conv2D', defined at:
  File "train_tacotron2.py", line 287, in <module>
    main()
  File "train_tacotron2.py", line 283, in main
    train(config.model_dir, config)
  File "train_tacotron2.py", line 132, in train
    loss_coeff=train_feeder.loss_coeff,stop_token_targets=train_feeder.stop_token_targets)
  File "/home/pirl/Tacotron2-Wavenet-Korean-TTS-master/tacotron2/tacotron2.py", line 82, in initialize
    x = tf.layers.conv1d(x,filters=hp.enc_conv_channels,kernel_size=hp.enc_conv_kernel_size,padding='same',activation=tf.nn.relu,name='Encoder_{}'.format(i))
.... 중략

UnknownError (see above for traceback): Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above.
	 [[node model/Encoder/Encoder_0/conv1d/Conv2D (defined at /home/pirl/Tacotron2-Wavenet-Korean-TTS-master/tacotron2/tacotron2.py:82) ]]
	 [[node model/optimizer/Adam/update (defined at /home/pirl/Tacotron2-Wavenet-Korean-TTS-master/tacotron2/tacotron2.py:298) ]]
..... 중략

2019-06-10 17:33:39.158518: W tensorflow/core/kernels/queue_base.cc:277] _0_datafeeder/input_queue_1: Skipping cancelled enqueue attempt with queue not closed
2019-06-10 17:33:39.158592: W tensorflow/core/kernels/queue_base.cc:277] _1_datafeeder/input_queue: Skipping cancelled enqueue attempt with queue not closed
... 중략

CancelledError (see above for traceback): Enqueue operation was cancelled
	 [[node datafeeder/input_queue_1_enqueue (defined at /home/pirl/Tacotron2-Wavenet-Korean-TTS-master/datasets/datafeeder_tacotron2.py:144) ]]
...중략

CancelledError (see above for traceback): Enqueue operation was cancelled
	 [[node datafeeder/input_queue_enqueue (defined at /home/pirl/Tacotron2-Wavenet-Korean-TTS-master/datasets/datafeeder_tacotron2.py:144) ]]

**Comment by hccho2** [comment]
cuda 설치 문제인 듯...
